docs/development/development-guide.md

```diff
@@ -0,0 +1,19 @@
+# Development Guide
+
+---
+
+This document walks you through how to get started with building KubeSphere in your local environment.
+
+## QuickStart
+
+---
+
+See [quickstart](quickstart.md)
+
+## Development Workflow
+
+See [development-workflow](development-workflow.md)
+
+## Pull Requests
+
+See [pull-requests](https://github.com/kubesphere/community/blob/master/developer-guide/development/pull-requests.md)
```

Comment:
it's better to put pull-requests.md in OpenFunction development docs:
https://github.com/kubesphere/community/blob/master/developer-guide/development/pull-requests.md